source/server-selection/server-selection.rst

```diff
@@ -351,9 +351,9 @@ serverSelector
 ~~~~~~~~~~~~~~
 
 Implementations MAY allow configuration of an optional, application-provided function
-that augments the server selection rules.  The function takes as a parameter a list
+that augments the server selection rules.  The function takes as a parameter an iterable
```

Comment:
I don't think we need to change this from list to iterable.  It's not like we're requiring any existing implementations to change their type signatures.

Reply:
So you are saying that in the spec "list" is to be interpreted loosely and means any platform specific type that is iterable?

It's certainly not going to be a `List<ServerDescrption>` in the C# driver.

It's an `IEnumerable<ServerDescription>` where `IEnumerable` is the C# name for `iterable`.

Reply:
Yes, list should be considered loosely.  Changing it to enumerable from list implies that we want drivers to make a change.  Also, list feels more generic to me and less like an an actual defined type (even though it is often a defined type).